sandbox/runx/impl/registry/registry.go

```diff
@@ -57,11 +125,20 @@ func (r *Registry) resolveVersion(ref types.PkgRef) (types.PkgRef, error) {
 		return types.PkgRef{}, err
 	}
 
-	if len(releases) == 0 {
+	found := false
+	latestVersion := "latest"
+	for _, release := range releases {
+		if !release.Draft && !release.Prerelease {
+			found = true
+			latestVersion = release.TagName
+			break
+		}
+	}
```

Comment:
If nothing is found, return pre-release?

Reply:
Done